tests/modules/apigee/env_only_with_api_proxy_type.yaml

```diff
@@ -14,16 +14,16 @@
 
 values:
   google_apigee_envgroup_attachment.envgroup_attachments["apis-test-test"]:
-    envgroup_id: test
+    envgroup_id: organizations/my-project/envgroups/test
```

Comment:
LGTM.
The test cases for `envgroup_id` were failing because of the proposed changes. Fixed the test cases.